src/transitiveClosure/transitiveClosure_driver.cpp

```diff
@@ -63,23 +63,23 @@ void get_postgres_result(
     int ii = 1;
     for (boost::tie(i, end) = boost::vertices(TC); i != end; ++i) {
             auto u = *i;
-            int64_t* target_array = NULL;
-            int64_t uid = graph.graph[u].id;
-            boost::tie(ai, a_end) = adjacent_vertices(*i, TC);
-            auto adj_siz = a_end - ai;
+            int64_t* target_array = nullptr;
+            auto uid = graph.graph[u].id;
+            boost::tie(ai, a_end) = adjacent_vertices(u, TC);
+            size_t adj_siz = static_cast<size_t>(a_end - ai);
```

Comment:
use `auto` so as to not duplicate the type name?

Reply:
There was auto, but auto failed on the rest of the code,
To making the change more visible I removed the auto.
when transitive closure code gets cleaned, those static_cast should disapear and come back to auto.
This PR objective is to remove warnings not to improve code